cli/Cargo.toml

```diff
@@ -19,7 +19,12 @@ nanorand = { version = "0.7.0", default-features = false, features = ["std", "wy
 rustyline = { version =  "12.0.1", default-features = false, features = ["with-file-history"], package = "rustyline-with-hint-fix" }
 serde = { version = "1.0.188", default-features = false }
 toml = { version = "0.7.6", default-features = false, features = ["parse"] }
-ureq = { version = "2.7.1", default-features = false, features = ["tls"] }
+ureq = { version = "2.7.1", default-features = false, optional = true }
```

Comment:
Is it correct to set ureq to be optional? I’m not currently at a computer to test this, but I assume this would cause fend to throw a compiler error which default features disabled.

Reply:
I set the default below: `default = [ "rustls" ]` , see line 30

Reply:
I've now pushed a commit that fixes the issue, it happened when running `cargo run --no-default-features`.